From ebd41fde5cf6ade113cbcd6fae35a192d06e8043 Mon Sep 17 00:00:00 2001
From: Allain Legacy <allain.legacy@windriver.com>
Date: Tue, 28 Nov 2017 15:13:33 -0600
Subject: [PATCH 094/155] fixup! db: fix notify data format for gateway ips

---
 neutron/db/l3_db.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/neutron/db/l3_db.py b/neutron/db/l3_db.py
index d9aca36..c5a1c24 100644
--- a/neutron/db/l3_db.py
+++ b/neutron/db/l3_db.py
@@ -496,7 +496,8 @@ class L3_NAT_dbonly_mixin(l3.RouterPluginBase,
                                                include_gateway=True)
             self._create_router_gw_port(context, router,
                                         new_network_id, ext_ips)
-            gw_ips = [x['ip_address'] for x in ext_ips or []]
+            gw_ips = [x['ip_address'] for x in ext_ips or []
+                      if 'ip_address' in x]
             registry.notify(resources.ROUTER_GATEWAY,
                             events.AFTER_CREATE,
                             self._create_gw_port,
-- 
2.7.4

