From 0b0489dd8c3a21bd2f27ab54d67a30d0cb23f95a Mon Sep 17 00:00:00 2001
From: Wen Shi <wen.shi@windriver.com>
Date: Thu, 1 Feb 2018 16:58:55 -0600
Subject: [PATCH 125/155] Update neutron config to support tox coverage

---
 tox.ini | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tox.ini b/tox.ini
index 43790a2..de7722a 100644
--- a/tox.ini
+++ b/tox.ini
@@ -134,8 +134,9 @@ whitelist_externals =
 [testenv:cover]
 basepython = python2.7
 commands =
+  coverage erase
   python setup.py testr --coverage --testr-args='{posargs}'
-  coverage report --fail-under=82 --skip-covered
+  coverage xml
 
 [testenv:venv]
 commands = {posargs}
-- 
2.7.4

